### ISSUE TITLE
plotly_build failed

### ISSUE THREAD

**Issue by hsiaoyi0504** [ropensci/plotly#816]
`ploty_build` can't reproduce some original figure on plotly.
```
library(plotly)
fig <- get_figure("b01303117", "1")
p <- plotly_build(fig)
```

For example, get figure and build this
[https://plot.ly/~b01303117/1/](https://plot.ly/~b01303117/1/)

 will produce 
```
Error in populate_categorical_axes(p) : 
  Can't display both discrete & non-discrete data on same axis
```

**Comment by cpsievert** [comment]
Thanks. There are few things going on here (some of which are fixed in 374330691a7ae76b280f56b712b9c1ee027e18f1)

For some reason the transform still doesn't render, but I'm not convinced this is at the fault of the R package. It seems to be that the plotly server isn't responding with the actual data to transform:

```r
> get_figure("b01303117", "1")$x$data[[2]]
$transforms
$transforms[[1]]
$transforms[[1]]$type
[1] "average"

$transforms[[1]]$inputUid
[1] "728216"

$transforms[[1]]$windowlength
[1] 5

$transforms[[1]]$center
[1] TRUE

$transforms[[1]]$windowtype
[1] "boxcar"



$name
[1] "mean=0.693%"

$uid
[1] "b7c0b4"
```

Any ideas @cldougl @tarzzz? If you look at the JSON at https://plot.ly/~b01303117/1 that seems to be incorrect as well

**Comment by cpsievert** [comment]
This doesn't appear to be a problem anymore
